features/promotion/applying_promotion_rules/receiving_discount_based_on_chosen_product.feature

```diff
@@ -0,0 +1,25 @@
+@applying_promotion_rules
+Feature: Receiving discount based on chosen product
+    In order to pay less while buying some goods
+    As a Customer
+    I want to receive discount for my purchase
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "PHP T-Shirt" priced at "$100.00"
+        And the store has a product "PHP Mug" priced at "$20.00"
+        And there is a promotion "T-Shirts promotion"
+
+    @ui
+    Scenario: Receiving discount on order while buying promoted product
+        Given the promotion gives "$20.00" off if order contains a "PHP T-Shirt" product
+        When I add product "PHP T-Shirt" to the cart
+        Then my cart total should be "$80.00"
+        And my discount should be "-$20.00"
+
```

Comment:
I think we're missing one more case, if I buy 2 units of "PHP T-Shirt" should I receive `-$20` discount or `-$40` discount?